kubernetes/cloud-config.yml

```diff
@@ -0,0 +1,22 @@
+azs:
+- name: z1
+
+vm_types:
+- name: default
+
+disk_types:
+- name: default
+  disk_size: 1024
+
+networks:
+- name: default
+  type: dynamic
+  subnets:
+  - azs: [z1]
+
+compilation:
+  workers: 3
```

Comment:
That was too much for my minikube installation. I can try out how many I could spin up in parallel. (GHC is quite a demanding compiler :) )